kuma/wiki/tests/test_views.py

```diff
@@ -2710,6 +2710,7 @@ def test_raw_include_option(self):
               <dd>Type: <em>integer</em></dd>
               <dd>Przykłady 例 예제 示例</dd>
             </dl>
+            <p><iframe></iframe></p>
```

Comment:
I was going to give feedback that the ``<iframe>`` test should be in it's own test, but 1) I don't want to make you convert to ``pytest`` as well, and 2) pyquery will return ``None`` if you just serialize an empty ``<iframe>``.